chapters/en-UK/xctest/types_of_testing.md

```diff
@@ -1,21 +1,23 @@
 # Types of Testing
 
-A test is a way of verifying code, and making your assumptions explicit. When you make a change somewhere, tests exist to show the implicit connections between objects.
```

Comment:
This tweet kind of sums up that idea for me - https://twitter.com/CodeWisdom/status/747469307022544897

There's a point where you're not aware of the full context for any change, tests help make those connections more implicit